package.json

```diff
@@ -18,8 +18,8 @@
     "prepublish": "npm run build",
     "prepublish-watch": "npm run watch",
     "start": "webpack-dev-server",
-    "tap": "./node_modules/.bin/tap ./test/unit/*.js",
-    "test": "npm run lint && npm run docs && npm run tap",
+    "tap": "./node_modules/.bin/tap ./test/unit/*.js test/integration",
+    "test": "npm run lint && npm run docs && npm run build && npm run tap",
```

Comment:
why is the build step in here? This is inconsistent with the use of the test script in the other repos.

Reply:
In order to do integration tests, we need to make sure the `dist/web/scratch-render.js` is up-to-date, I feel like anything doing integration tests likely does a build before the tests?

Reply:
Yeah, but usually not in the `test` script itself, e.g. see `scratch-vm`